pkg/cli/subcommands/check/command.go

```diff
@@ -230,6 +232,7 @@ func MakeCommand(globalParamsGetter func() GlobalParams) *cobra.Command {
 	cmd.Flags().UintVarP(&cliParams.discoveryTimeout, "discovery-timeout", "", 5, "max retry duration until Autodiscovery resolves the check template (in seconds)")
 	cmd.Flags().UintVarP(&cliParams.discoveryRetryInterval, "discovery-retry-interval", "", 1, "(unused)")
 	cmd.Flags().UintVarP(&cliParams.discoveryMinInstances, "discovery-min-instances", "", 1, "minimum number of config instances to be discovered before running the check(s)")
+	cmd.Flags().StringVarP(&cliParams.customConfig, "custom-config", "C", "", "pass custom check config")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&cliParams.customConfig, "custom-config", "C", "", "path to a custom check configuration file")
```
The description could use more clarification